inventory/services_test.go

```diff
@@ -1210,7 +1210,7 @@ func TestExternalService(t *testing.T) {
 		})
 		assert.NoError(t, err)
 		assert.NotNil(t, noFilterServicesList)
-		assert.Len(t, noFilterServicesList.Payload.Mysql, 1)
+		assert.Len(t, noFilterServicesList.Payload.Mysql, 0)
```

Comment:
@askomorokhov This test was broken before? Or please what changed here? Other changes look good.

Reply:
Looks like it was broken before. I saw in some PR it was changed - but did not pay attention enough to it.